.github/workflows/web-test.yaml

```
@@ -0,0 +1,118 @@
+
+      - name: Setup Chrome and ChromeDriver
+        run: |
+          # Install Chrome
+          sudo install -m 644 <(curl -fsSL https://dl.google.com/linux/linux_signing_key.pub) /usr/share/keyrings/google-linux-signing-keyring.gpg
+          echo "deb [arch=amd64 signed-by=/usr/share/keyrings/google-linux-signing-keyring.gpg] \
+                http://dl.google.com/linux/chrome/deb/ stable main" | \
+                sudo tee /etc/apt/sources.list.d/google-chrome.list
+
+          sudo apt-get update
+          sudo apt-get install -y google-chrome-stable jq
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)